esignet-core/src/main/java/io/mosip/esignet/core/dto/PAROAuthDetailsRequest.java

```diff
@@ -0,0 +1,17 @@
+package io.mosip.esignet.core.dto;
+
+import io.mosip.esignet.core.constants.ErrorConstants;
+import io.mosip.esignet.core.validator.RequestUri;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+public class PAROAuthDetailsRequest {
```

Comment:
Existing class is named as OAuthDetailRequest.java. I think we should name the class as PushedOAuthDetailRequest.java